flutter/README.md

```diff
@@ -0,0 +1,84 @@
+# fluentui_system_icons
+
+![CI](https://github.com/microsoft/fluentui-system-icons/workflows/CI/badge.svg)
+
+Fluent System Icons is a flutter package for a set of mobile platform icons from Microsoft.
+
+![fluent system icons](../art/readme-asset.png)
+
+## Icon List
+
+[View the full list of icons](../icons.md)
+
+### Icon styles
+The library offers icons in two styles, `regular` and `filled`
+
+regular | filled
+--------|-------
+![ic_fluent_mail_regular](../art/ic_fluent_mail_regular.png)|![ic_fluent_mail_filled](../art/ic_fluent_mail_filled.png)
```

Comment:
This brings up a great question. @jasoncuster do we have a "default" or representative icon you would suggest be used here?

Reply:
I think this is fine, but we could update it to use the SVGs from the assets folder since it's supported by Github markdown. Examples can be found in https://github.com/microsoft/fluentui-system-icons/blob/master/icons.md